crypto/libring.Windows.vcxproj

```diff
@@ -60,6 +60,7 @@
     <ClCompile>
       <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);..\include</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalOptions>/source-charset:utf-8</AdditionalOptions>
```

Comment:
Thanks! Should we use ` /utf-8` instead? In particular, it seems like we want to set the execution charset to UTF-8 to, if I'm understanding the documentation correctly. (The execution charset probably doesn't matter.)